pkg/services/ngalert/models/alert_rule.go

```diff
@@ -288,6 +301,16 @@ type AlertRuleGroupKey struct {
 	RuleGroup    string
 }
 
+func (k AlertRuleGroupKey) Less(v AlertRuleGroupKey) bool {
```

Comment:
Is this the sorting that the front-end expects? This is sorting by `IDs` as opposed to alphabetically, which is what the front-end currently expects.

Reply:
Where do you see `ID` Josh? Here `AlertRuleGroupKey` sorts on Org, Namespace UID and then Rule Group?

Reply:
Where do you see `ID` Josh? Here `AlertRuleGroupKey` sorts on Org ID, Namespace UID and then Rule Group?

Reply:
I have resolved my own comment here.